build.sbt

```diff
@@ -126,112 +123,119 @@ lazy val openapiCirceYaml: Project = (project in file("openapi/openapi-circe-yam
 // docs
 
 lazy val openapiDocs: Project = (project in file("docs/openapi-docs"))
-  .settings(commonSettings: _*)
+  .settings(commonSettings(): _*)
   .settings(
     name := "tapir-openapi-docs"
   )
   .dependsOn(openapiModel, core, tests % "test", openapiCirceYaml % "test")
 
 lazy val swaggerUiAkka: Project = (project in file("docs/swagger-ui-akka-http"))
-  .settings(commonSettings: _*)
+  .settings(commonSettings(): _*)
   .settings(
     name := "tapir-swagger-ui-akka-http",
     libraryDependencies ++= Seq(
-      "com.typesafe.akka" %% "akka-http" % akkaHttpVersion,
-      "com.typesafe.akka" %% "akka-stream" % akkaStreamsVersion,
-      "org.webjars" % "swagger-ui" % swaggerUiVersion
+      "com.typesafe.akka" %% "akka-http" % Versions.akkaHttp,
+      "com.typesafe.akka" %% "akka-stream" % Versions.akkaStreams,
+      "org.webjars" % "swagger-ui" % Versions.swaggerUi
     )
   )
 
 lazy val swaggerUiHttp4s: Project = (project in file("docs/swagger-ui-http4s"))
-  .settings(commonSettings: _*)
+  .settings(commonSettings(): _*)
   .settings(
     name := "tapir-swagger-ui-http4s",
-    libraryDependencies ++= Seq(
-      "org.http4s" %% "http4s-dsl" % http4sVersion,
-      "org.webjars" % "swagger-ui" % swaggerUiVersion
+    libraryDependencies ++= dependenciesFor(scalaVersion.value)(
+      "org.http4s" %% "http4s-dsl" % Versions.http4s(_),
+      _ => "org.webjars" % "swagger-ui" % Versions.swaggerUi
     )
   )
 
 // server
 
 lazy val serverTests: Project = (project in file("server/tests"))
-  .settings(commonSettings: _*)
+  .settings(commonSettings(Seq(scala2_12)): _*)
   .settings(
     name := "tapir-server-tests",
     publishArtifact := false,
-    libraryDependencies ++= Seq("com.softwaremill.sttp" %% "async-http-client-backend-cats" % sttpVersion)
+    libraryDependencies ++= (CrossVersion.partialVersion(scalaVersion.value) match {
+      case Some((2, 12)) =>
+        Seq("com.softwaremill.sttp" %% "async-http-client-backend-cats" % Versions.sttp)
```

Comment:
there is no async-http-client-backend-cats for 2.13 yet. What should I do here?

Reply:
I copied your approach to sttp and released 1.6.3 with almost all modules for 2.13 (except monix & circe-js): https://github.com/softwaremill/sttp/commit/8457960c837aed8a73154e5845bd4ebf7d2aa633

So I think this should help? :)

Reply:
great.